sample/Droidcon17/event

```diff
@@ -0,0 +1,77 @@
+{
+	"id": 182,
+	"name": "Droidcon 2017",
+	"latitude": "",
+	"longitude": "",
+	"location_name": "Calderwood Pavilion, 527 Tremont St, Boston, MA 02116",
```

Comment:
@Shailesh351 since the new format hasn't been fully implemented I am continuing to use the old one. Once it is done, I will probably use script made by @arp95 to convert them, or make my own if they don't work. 👍 
Please point out if there's anything else.